ui/ui-components/pages/app/[id]/refresh.tsx

```diff
@@ -113,11 +118,17 @@ export default function Page(props) {
         );
         if (response?.valueUpdated == true) {
           successHandler.set({
-            message: `App Refresh Query updated. Returned ${response.appRefreshQuery.value}. Enqueueing Schedules.`,
+            message: `App Refresh Query updated. Returned ${
+              response.appRefreshQuery.value
+            }.${grouparooUiEdition() !== "config" ?? " Enqueueing Schedules"}`,
```

Comment:
This may have two spaces in the text if we're not in config mode looks like.

Reply:
Can you help me understand that?  My intention was that config mode would return something like
`App Refresh Query updated. Returned {date: 1234567890}.`
Whereas non-config modes would return ``App Refresh Query updated. Returned {date: 1234567890}. Enqueueing schedules` (or not enqueueing schedules).

Are you saying that if we're in config mode, what I wrote would put a space where the second statement is instead of just an empty string?